### ISSUE TITLE
gatsby new hello-world error

### ISSUE THREAD

**Issue by zwl1619** [gatsbyjs/gatsby#24749]
```
D:\workspace\www\gatsby_blog
λ gatsby new hello-world https://github.com/gatsbyjs/gatsby-starter-hello-world
//...
npm WARN tar ENOENT: no such file or directory, open 'D:\workspace\www\gatsby_blog\hello-world\node_modules\.staging\gatsby-0c654513\dist\schema\types\derived-types.js'
npm WARN tar ENOENT: no such file or directory, open 'D:\workspace\www\gatsby_blog\hello-world\node_modules\.staging\gatsby-recipes-80ff5330\src\providers\npm\__snapshots__\script.test.js.snap'
npm WARN tar ENOENT: no such file or directory, open 'D:\workspace\www\gatsby_blog\hello-world\node_modules\.staging\gatsby-0c654513\cache-dir\__tests__\dev-loader.js'
npm WARN tar ENOENT: no such file or directory, open 'D:\workspace\www\gatsby_blog\hello-world\node_modules\.staging\gatsby-0c654513\cache-dir\commonjs\dev-loader.js'
npm WARN tar ENOENT: no such file or directory, open 'D:\workspace\www\gatsby_blog\hello-world\node_modules\.staging\gatsby-0c654513\cache-dir\dev-loader.js'
npm WARN tar ENOENT: no such file or directory, open 'D:\workspace\www\gatsby_blog\hello-world\node_modules\.staging\gatsby-0c654513\dist\commands\develop-process.js'
npm ERR! Maximum call stack size exceeded

npm ERR! A complete log of this run can be found in:
npm ERR!     C:\Users\admin\AppData\Roaming\npm-cache\_logs\2020-06-03T12_11_37_982Z-debug.log

 ERROR

Command failed with exit code 1: npm install



  Error: Command failed with exit code 1: npm install

  - error.js:56 makeError
    [node]/[gatsby-cli]/[execa]/lib/error.js:56:11

  - index.js:114 handlePromise
    [node]/[gatsby-cli]/[execa]/index.js:114:26

  - task_queues.js:97 processTicksAndRejections
    internal/process/task_queues.js:97:5

  - init-starter.js:139 async install
    [node]/[gatsby-cli]/lib/init-starter.js:139:7

  - init-starter.js:206 async clone
    [node]/[gatsby-cli]/lib/init-starter.js:206:3

  - init-starter.js:345 async initStarter
    [node]/[gatsby-cli]/lib/init-starter.js:345:19

  - create-cli.js:400
    [node]/[gatsby-cli]/lib/create-cli.js:400:7
```

**Comment by vladar** [comment]
Looks like `npm` install was not successful in your case (see node is looking for modules in the `.staging` folder)

You can try the following (inside `D:\workspace\www\gatsby_blog\hello-world`):

1. Delete `package-lock.json`
2. Delete `node_modules` folder
3. Run `npm install`

If this doesn't help then you should try upgrading `nodejs`. 
Also, you can try debugging it this way:

1. Run `npm audit` (shows the list of pending packages which probably failed to install)
2. Run `npm install [packagename]` (where `[packagename]` is the name of the package from audit list above. Run this for each failing package and check for errors)

So to recap: this problem has something to do with a bad setup of nodejs/npm on your machine or failing `npm install`
